lib/Doctrine/Common/Cache/MemcachedCache.php

```diff
@@ -92,6 +92,16 @@ protected function doSaveMultiple(array $keysAndValues, $lifetime = 0)
     /**
      * {@inheritdoc}
      */
+    protected function doDeleteMultiple(array $keys)
+    {
+        $stat = $this->memcached->deleteMulti($keys);
+        return count(array_unique($stat) > 1)
+            || end($stat) === true;
```

Comment:
Yes, that would require all operations to be successful. Like I've said, it can easily be changed. I just wasn't sure what the preferred behavior would be. 

For example, if you look at `RedisCache`, the normal `doDelete` call returns `true` even if no entry was affected. The logic you propose would have to return `false` in such a case, causing an inconsistency between drivers. On the other hand, driver behavior doesn't look that consistent to begin with afaict, so I'll take whichever version you prefer :-)